easybuild/easyconfigs/h/horton/horton-2.1.1-foss-2018b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'horton'
+version = '2.1.1'
+
+homepage = 'http://theochem.github.io/horton'
+description = """HORTON is a Helpful Open-source Research TOol for N-fermion systems, written
+ primarily in the Python programming language. (HORTON is named after the helpful
+ pachyderm, not the Canadian caffeine supply store.) The ultimate goal of HORTON
+ is to provide a platform for testing new ideas on the quantum many-body problem
+ at a reasonable computational cost. Although HORTON is primarily designed to be
+ a quantum-chemistry program, it can perform computations involving model
+ Hamiltonians, and could be extended for computations in nuclear physics."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/theochem/horton/releases/download/%s' % version]
+sources = [SOURCE_TAR_GZ]
+checksums = ['13e3381dea62e5877c6cde94515e99c8b7872f0a9498961b0766bdeae4506fe6']
+
+pyver = '2.7.15'
+pyversuffix = '-Python-%s' % pyver
+dependencies = [
+    ('Python', pyver),
+    ('h5py', '2.8.0', pyversuffix),
+    ('matplotlib', '2.2.3', pyversuffix),
+    ('Libint', '2.0.3'),
+    ('libxc', '2.2.3'),
+]
+
+prebuildopts = ' '.join([
+    'BLAS_EXTRA_COMPILE_ARGS=-DMKL_ILP64:-I${MKLROOT}/include',
+    'BLAS_LIBRARY_DIRS=${MKLROOT}/lib/intel64',
+    'BLAS_LIBRARIES=mkl_intel_ilp64:mkl_core:mkl_sequential:pthread:m:mkl_def',
+])
+
```

Comment:
@tovrstra In the existing easyconfig for `horton`, we also run the tests, as follows:

```python
# Avoid need for X11 in tests by specifying "backend: agg" in matplotlibrc
runtest = ' '.join([
    'export MATPLOTLIBRC=$PWD;',
    'echo "backend: agg" > $MATPLOTLIBRC/matplotlibrc;',
    '%s python setup.py build_ext -i; ' % prebuildopts,
    'nosetests -v',
])
```